### PR TITLE
Prefer channels for installing different versions

### PR DESCRIPTION
**Pull request by evilnick**
As we now have enough channels in charmhub to cover all the supported versions, remove the 'revision number' methods of install and point users to the repo for older bundle files.

We may need to revisit the autogeneration tools in a separate PR

### CODE REVIEW

File: pages/k8s/install-offline.md [hunk 1]
```diff
@@ -212,28 +212,31 @@ from an internet connected machine:
...
-cd cs:charmed-kubernetes-862-stable-*/
-./deploy.sh
-# examine provided instructions
-# ensure necessary modifications are considered
-juju deploy ...
-```
+1. Finally, deploy the Juju charms and resources from the provided local bundle:
+
+   ```bash
+   tar -xvf charmed-kubernetes-*.tar.gz --force-local
```
Comment (evilnick) [inline pages/k8s/install-offline.md:231]: i *think* this works but will be able to check the filename when i have finished downloading the internet

File: pages/k8s/.markdownlint.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "MD033": false
```
Comment (kwmonroe) [inline pages/k8s/.markdownlint.json:2]: TIL about markdownlint!

File: pages/k8s/install-manual.md [hunk 3]
```diff
@@ -228,31 +224,10 @@ This also applies to channels which have not yet had a stable release, for examp
...
-| 1.21.x    | [charmed-kubernetes-733](https://raw.githubusercontent.com/charmed-kubernetes/bundle/main/releases/1.21/bundle.yaml) |
-| 1.20.x    | [charmed-kubernetes-596](https://raw.githubusercontent.com/charmed-kubernetes/bundle/main/releases/1.20/bundle.yaml) |
-| 1.19.x    | [charmed-kubernetes-545](https://raw.githubusercontent.com/charmed-kubernetes/bundle/main/releases/1.19/bundle.yaml) |
-| 1.18.x    | [charmed-kubernetes-485](https://raw.githubusercontent.com/charmed-kubernetes/bundle/main/releases/1.18/bundle.yaml) |
-| 1.17.x    | [charmed-kubernetes-410](https://raw.githubusercontent.com/charmed-kubernetes/bundle/main/releases/1.17/bundle.yaml) |
-| 1.16.x    | [charmed-kubernetes-316](https://raw.githubusercontent.com/charmed-kubernetes/bundle/main/releases/1.16/bundle.yaml) |
-| 1.15.x    | [charmed-kubernetes-209](https://raw.githubusercontent.com/charmed-kubernetes/bundle/main/releases/1.15/bundle.yaml) |
-| 1.14.x    | [charmed-kubernetes-124](https://raw.githubusercontent.com/charmed-kubernetes/bundle/main/releases/1.14/bundle.yaml) |
-<!-- AUTOGENERATED BUNDLE TABLE END  -->
```
Comment (kwmonroe) [inline pages/k8s/install-manual.md:255]: good riddance

**Review by kwmonroe — APPROVED** [review]
Nice job.

:cake:

File: pages/k8s/.markdownlint.json (hunk 2 above)
Comment (evilnick) [reply pages/k8s/.markdownlint.json:2]: yeah, that day i learnt how to get rid of the warnings i don't want to see!

**Pull request merged**